Added documents for application access policy cmdlets (#6464)
MicrosoftDocs/office-docs-powershell: skype/skype-ps/skype/Grant-CsApplicationAccessPolicy.md

```

The command shown above assigns the per-user application access policy "ASimplePoicy" to all the users in the tenant, except any that have an explicit policy assignment.

## PARAMETERS

### -Identity

Indicates the user (object) ID of the user account to be assigned the per-user application access policy. 

```yaml
Type: UserIdParameter
Parameter Sets: (All)
Aliases: 

Required: True
Position: 0
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -PolicyName

Name of the policy to be assigned. The PolicyName is simply the policy Identity minus the policy scope (the "tag:" prefix). For example, a policy with the Identity tag:ASimplePolicy has a PolicyName equal to ASimplePolicy.

```yaml
Type: PSListModifier
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -Global

When you use this cmdlet without specifying a user identity, the policy applies to all users in your tenant, except any that have an explicit policy assignment. To skip a warning when you do this operation, specify this parameter.

```yaml
Type: SwitchParameter

> Parameter Sets: GrantToTenant

Reviewer: This parameter set does not appear in the output of Platy nor the Sintax section, should we put Parameter Sets: (All) ?

Author: updated

---
external help file: Microsoft.Rtc.Management.dll-help.xml
online version: https://docs.microsoft.com/powershell/module/skype/grant-csapplicationaccesspolicy
applicable: Skype for Business Online
title: Grant-CsApplicationAccessPolicy
schema: 2.0.0
manager: zhengni
author: frankpeng7
ms.author: frpeng
ms.reviewer:
---

# Grant-CsApplicationAccessPolicy

## SYNOPSIS

Assigns a per-user application access policy to one or moer users. After assigning an application access policy to a user, the applications configured in th policy will be authorized to access online meetings on behalf of that user.

## SYNTAX

### FileName

```

> Grant-CsApplicationAccessPolicy [-Identity <UserIdParameter>] [-PolicyName <String>]

Reviewer 2: ```suggestion
Grant-CsApplicationAccessPolicy [-Identity <UserIdParameter>] [-PolicyName <String>] [-Global]
```

```

## DESCRIPTION

<TODO: description>

## EXAMPLES

### Assign an application access poicy to a user

```
PS C:\> Grant-CsApplicationAccessPolicy -Identity "dc17674c-81d9-4adb-bfb2-8f6a442e4624" -PolicyName "ASimplePolicy"
```

The command shown above assigns the per-user application access policy "ASimplePolicy" to the user with obect ID "dc17674c-81d9-4adb-bfb2-8f6a442e4624".

> ### UnAssign an application access poicy from a user

Reviewer 2: ```suggestion
### Unassign an application access policy from a user
```

```
PS C:\> Grant-CsApplicationAccessPolicy -Identity "dc17674c-81d9-4adb-bfb2-8f6a442e4624" -PolicyName $Null
```

> In the command shown above , any per-user application access policy previously assigned to the user with user (object) ID "dc17674c-81d9-4adb-bfb2-8f6a442e4624" is unassigned from that user; as a result, applications configured in the policy can no longer access online meetings on behalf of that user. To unassign a per-user policy, set the PolicyName to a null value ($Null).

Reviewer 2: ```suggestion
In the command shown above, any per-user application access policy previously assigned to the user with user (object) ID "dc17674c-81d9-4adb-bfb2-8f6a442e4624" is unassigned from that user; as a result, applications configured in the policy can no longer access online meetings on behalf of that user. To unassign a per-user policy, set the PolicyName to a null value ($Null).
```

### Assign an application access policy to all users in the tenant

```
PS C:\> Get-CsOnlineUser | Grant-CsApplicationAccessPolicy -PolicyName "ASimplePolicy"
```

The command shown above assigns the per-user application access policy ASimplePolicy to all the users in the tenant. To do this, the command first calls the `Get-CsOnlineUser` cmdlet to get all user accounts enabled for Skype for Business Online. Those user accounts are then piped to the `Grant-CsApplicationAccessPolicy` cmdlet, which assigns each user the application access policy "ASimplePolicy".

### Assign an application access policy to users who have not been assigned one

```
PS C:\> Grant-CsApplicationAccessPolicy -PolicyName "ASimplePolicy" -Global
```

The command shown above assigns the per-user application access policy "ASimplePoicy" to all the users in the tenant, except any that have an explicit policy assignment.

## PARAMETERS

### -Identity

> Indicates the user (object) ID of the user account to be assigned the per-user online voice routing policy. 

Reviewer 2: ```suggestion
Indicates the user (object) ID of the user account to be assigned the per-user application access policy. 
```

```yaml
Type: UserIdParameter

> Parameter Sets: Identity

Reviewer 2: ```suggestion
Parameter Sets: (All)
```

Aliases: 

Required: True
Position: 0
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -PolicyName

> <TODO: description for parameter?> "Name" of the policy to be assigned. The PolicyName is simply the policy Identity minus the policy scope (the "tag:" prefix). For example, a policy with the Identity tag:ASimplePolicy has a PolicyName equal to ASimplePolicy.

Reviewer 2: ```suggestion
Name of the policy to be assigned. The PolicyName is simply the policy Identity minus the policy scope (the "tag:" prefix). For example, a policy with the Identity tag:ASimplePolicy has a PolicyName equal to ASimplePolicy.
```

```yaml

> Type: PSListModifier

Reviewer 2: ```suggestion
Type: String
```

Author: I actually believe this should be a list

Reviewer 2: But, is it one policy that you grant to the user? Or multiple policies?

Author: one policy, but this parameter takes a list (of strings)
